gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/CopyJobConfiguration.java

```diff
@@ -31,7 +31,7 @@
  * Google BigQuery copy job configuration. A copy job copies an existing table to another new or
  * existing table. Copy job configurations have {@link JobConfiguration.Type#COPY} type.
  */
-public final class CopyJobConfiguration extends JobConfiguration {
+public class CopyJobConfiguration extends JobConfiguration {
```

Comment:
I think this class should be an instance of "final class, non-final equals/hashcode" since this class has a builder with only public setters.